src/mono/sample/Android/AndroidSampleApp.csproj

```diff
@@ -18,6 +18,9 @@
     <RepoDotnet Condition="'$(DOTNET_ROOT)' != ''">$(DOTNET_ROOT)/dotnet</RepoDotnet>
     <RepoDotnet Condition="'$(RepoDotnet)' == '' and '$(OS)' == 'Windows_NT'">..\..\..\..\dotnet.cmd</RepoDotnet>
     <RepoDotnet Condition="'$(RepoDotnet)' == '' and '$(OS)' != 'Windows_NT'">../../../dotnet.sh</RepoDotnet>
+    <!-- R2R -->
+    <!-- <PublishReadyToRun>true</PublishReadyToRun> -->
```

Comment:
Lets move this as an option that can be passed from the Makefile https://github.com/dotnet/runtime/blob/42423ef91c9f23381d99dee708fa55336318debc/src/mono/sample/Android/Makefile

Reply:
Since we can build on all platforms, should we move away from makefiles?

Reply:
Made it configurable via passing `R2R` and `R2R_COMPOSITE` (defaults to false).